v0.14.0 panic with SIGSEGV

After auto-upgrading to the latest version of statsd_exporter, my statsd_exporter deployment in kubernetes began crashing. Here's the log output:

```
level=info ts=2020-01-10T21:36:34.923Z caller=main.go:170 msg="Starting StatsD -> Prometheus Exporter" version="(version=0.14.0, branch=HEAD, revision=20006621cc9de5de35b71fea9d0e5e12f075ccd5)"
level=info ts=2020-01-10T21:36:34.923Z caller=main.go:171 msg="Build context" context="(go=go1.13.6, user=XXX@XXX, date=20200110-17:02:47)"
level=info ts=2020-01-10T21:36:34.923Z caller=main.go:172 msg="Accepting StatsD Traffic" udp=:8125 tcp=:8125 unixgram=
level=info ts=2020-01-10T21:36:34.923Z caller=main.go:173 msg="Accepting Prometheus Requests" addr=:9121
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x85c997]

goroutine 58 [running]:
github.com/go-kit/kit/log.(*context).Log(0xc000182870, 0xc000334060, 0x6, 0x6, 0x2, 0xa91280)
	/app/vendor/github.com/go-kit/kit/log/log.go:124 +0x207
main.parseTag(0xc00014611b, 0x5f, 0xc000146143, 0xb, 0xc00000003a, 0xc000182810, 0x0, 0x0)
	/app/exporter.go:309 +0x598
main.parseDogStatsDTags(0xc00014611b, 0x5f, 0xc000182810, 0x0, 0x0)
	/app/exporter.go:342 +0xf0
main.lineToEvents(0xc000146100, 0x7a, 0x0, 0x0, 0x0, 0x0, 0xc0000f4b00)
	/app/exporter.go:454 +0x11d8
main.(*StatsDUDPListener).handlePacket(0xc000182a50, 0xc00024ff91, 0x7a, 0xffff)
	/app/exporter.go:512 +0x13c
main.(*StatsDUDPListener).Listen(0xc000182a50)
	/app/exporter.go:503 +0x8a
created by main.main
	/app/main.go:202 +0x3a2d
```

I've downgraded to v0.13.0 and that has resolved things for now.

--- a/main.go
+++ b/main.go
@@ -198,7 +198,7 @@ func main() {
 			}
 		}
 
-		ul := &StatsDUDPListener{conn: uconn, eventHandler: eventQueue}
+		ul := &StatsDUDPListener{conn: uconn, eventHandler: eventQueue, logger: logger}
 		go ul.Listen()
 	}
 

